bin/utils/prompt-command.js

```diff
@@ -53,7 +56,15 @@ module.exports = function promptForInstallation(packages, ...args) {
 	try {
 		const path = require("path");
 		const fs = require("fs");
-		pathForCmd = path.resolve(process.cwd(), "node_modules", "@webpack-cli", packages);
+		const findup = require("findup-sync");
```

Comment:
You cannot use the findProj root function for this. This should exactly match cwd as we cannot traverse up directories to search for a project. We need to assume that the user is in the current folder. Otherwise, people might install webpack-cli at /Desktop instead of /Desktop/myproj or something.